homeassistant/components/oru/sensor.py

```diff
@@ -0,0 +1,92 @@
+"""Platform for sensor integration."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from oru import Meter
+from oru import MeterError
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import ENERGY_KILO_WATT_HOUR
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_METER_NUMBER = "meter_number"
+
+SCAN_INTERVAL = timedelta(minutes=15)
+
+SENSOR_NAME = "ORU Current Energy Usage"
+SENSOR_ICON = "mdi:counter"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({vol.Required(CONF_METER_NUMBER): cv.string})
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the sensor platform."""
+
+    meter_number = str(config.get(CONF_METER_NUMBER, None))
+
+    try:
+        meter = Meter(meter_number)
+
+    except MemoryError:
+        _LOGGER.error("Unable to create Oru meter")
+        return
+
+    add_entities([CurrentEnergyUsageSensor(meter)])
+
+    _LOGGER.debug("meter_number = %s", meter_number)
```

Comment:
We start logging messages with capital letter.

Maybe we don't need to log the config option? We should know what we entered in the config.

Reply:
Done. Started with capital letter. I'd like to leave this in, just in case. It's` debug` anyway.